src/public/js/index.js

```diff
@@ -0,0 +1,28 @@
+import getTimeRemaining from './countdown';
+
+const initializeClock = (minutes) => {
+  document.getElementById('start').style.display = 'none';
+  document.getElementById('stop').style.display = 'block';
```

Comment:
What if the div display isn't block or we want to change it from a block, we won't have the flexibility to change it.

Could you hide and show this without restricting the div to display block